generator/cli/kallax/cmd.go

```diff
@@ -69,10 +88,44 @@ func generateModels(c *cli.Context) error {
 	return nil
 }
 
+func migrateAction(c *cli.Context) error {
+	dirs := c.StringSlice("input")
+	dir := c.String("out")
+	name := c.String("name")
+
+	var pkgs []*generator.Package
+	for _, dir := range dirs {
+		if !isDirectory(dir) {
+			return fmt.Errorf("kallax: `input` must be a valid directory")
+		}
+
+		p := generator.NewProcessor(dir, nil)
+		p.Silent()
+		pkg, err := p.Do()
+		if err != nil {
+			return err
+		}
+
+		pkgs = append(pkgs, pkg)
+	}
+
+	if !isDirectory(dir) {
+		return fmt.Errorf("kallax: `out` must be a valid directory")
+	}
+
+	g := generator.NewMigrationGenerator(name, dir)
+	migration, err := g.Build(pkgs...)
+	if err != nil {
+		return err
+	}
+
+	return g.Generate(migration)
+}
+
 func isDirectory(name string) bool {
 	info, err := os.Stat(name)
 	if err != nil {
-		log.Fatal(err)
+		return false
```

Comment:
There are some cases that the path will be a directory but `os.Stat()` will return an error, per example in permission errors.